conversion_test.go

```diff
@@ -211,7 +223,10 @@ func TestFromBigOverflow(t *testing.T) {
 }
 
 func TestToBig(t *testing.T) {
-
+	var uint256Nil *Int
+	if bigNil := uint256Nil.ToBig(); bigNil != nil {
+		t.Errorf("expected big.Int <nil>, got %x", bigNil)
```

Comment:
```suggestion
		t.Errorf("want big.Int <nil>, have %x", bigNil)
```
nit just to trigger CI